receiver/azureeventhubreceiver/client.go

```diff
@@ -124,16 +127,9 @@ func (c *client) setUpOnePartition(ctx context.Context, partitionID string, appl
 
 func (c *client) handle(ctx context.Context, event *eventhub.Event) error {
 	c.obsrecv.StartLogsOp(ctx)
-	l := plog.NewLogs()
-	lr := l.ResourceLogs().AppendEmpty().ScopeLogs().AppendEmpty().LogRecords().AppendEmpty()
-	slice := lr.Body().SetEmptyBytes()
-	slice.Append(event.Data...)
-	lr.Attributes().FromRaw(event.Properties)
-	if event.SystemProperties.EnqueuedTime != nil {
-		lr.SetTimestamp(pcommon.NewTimestampFromTime(*event.SystemProperties.EnqueuedTime))
-	}
-	consumerErr := c.consumer.ConsumeLogs(ctx, l)
-	c.obsrecv.EndLogsOp(ctx, "azureeventhub", 1, consumerErr)
+	logs, _ := c.convert.ToLogs(event)
```

Comment:
we should not ignore the error here

Reply:
Error captured and returned. Other receivers are not consistent on what to do with the start/end op calls. I've moved it after the conversion to avoid having an unfinished start/end pair of calls.